packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/DrawingTask/DrawingTask.js

```diff
@@ -1,56 +1,101 @@
-import { Markdownz } from '@zooniverse/react-components'
 import { Text } from 'grommet'
+import { Blank } from 'grommet-icons'
 import { observable } from 'mobx'
 import { inject, observer, PropTypes as MobXPropTypes } from 'mobx-react'
-import React from 'react'
 import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import { Markdownz } from '@zooniverse/react-components'
+import getIcon from './helpers/getIcon'
+import InputIcon from '../InputIcon'
+import InputStatus from '../InputStatus'
+import TaskInputField from '../TaskInputField'
+
+export const StyledFieldset = styled.fieldset`
+  border: none;
+  margin: 0;
+  padding: 0;
+`
+
+const ToolIcon = ({ type }) => {
+  const Icon = getIcon(type)
+  return (
+    <Blank viewBox='0 0 100 100'>
+      <Icon />
+    </Blank>
+  )
+}
+
+const StyledText = styled(Text)`
+  margin: 0;
+  padding: 0;
+  width: 100%;
+
+  > *:first-child {
+    margin-top: 0;
+  }
+`
 
 function storeMapper (stores) {
-  const {
-    addAnnotation
-  } = stores.classifierStore.classifications
-  const annotations = stores.classifierStore.classifications.currentAnnotations
+  const { active, setActive } = stores.classifierStore.drawing
   return {
-    addAnnotation,
-    annotations
+    active,
+    setActive
   }
 }
 
 @inject(storeMapper)
 @observer
 class DrawingTask extends React.Component {
   onChange (index, event) {
-    const { addAnnotation, task } = this.props
-    if (event.target.checked) addAnnotation(index, task)
+    const { setActive } = this.props
+    if (event.target.checked) {
+      setActive(index)
+    }
   }
 
   render () {
     const {
-      annotations,
+      active,
       task
     } = this.props
-    // let annotation
-    // if (annotations && annotations.size > 0) {
-    //   annotation = annotations.get(task.taskKey)
-    // }
     return (
-      <Text size='small' tag='legend'>
-        <Markdownz>
-          {task.instruction}
-        </Markdownz>
-      </Text>
+      <StyledFieldset>
```

Comment:
Discussion started in #1020:
@eatyourgreens:
I don't think Fieldset is the right element to use here. Fieldsets have simple, inline text legends but our task questions are block-level elements, sometimes with more than one paragraph.

I think this should just be a Grommet Box.

eatyourgreens:
Fieldset/Legend can't take block-level children, so can't wrap a Markdownz component.

eatyourgreens:
Here's the list of what's valid in a fieldset legend. I don't think we can easily switch to using these elements for task questions without breaking existing live workflows.
https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content

@srallen:
This was my bad when first starting on the task area and I think it's being copied around. I was thinking of using valid form elements if in theory we could have a PWA that could work without javascript, but since there's markdown going here, we won't be able to guarantee the question is valid HTML for these elements.

eatyourgreens:
We could possibly mark up transcription tasks as forms that submit the values of the text inputs, or question tasks as forms that submit the answers. Drawing tasks would submit the value of the selected drawing tool, so I'm thinking either we move the form markup down to specific types of task, or drop it completely.

Reply:
Addressed in #1039